src/screens/Signup/StepCaptcha/index.tsx

```diff
@@ -50,6 +52,7 @@ export function StepCaptcha() {
         type: 'setError',
         value: _(msg`Error receiving captcha response.`),
       })
+      logEvent('signup:captchaFailure', {})
       logger.error('Signup Flow Error', {
```

Comment:
Odd that we don't have any errors in Sentry for this

Reply:
Oh you know I probably filtered this out when we were getting slammed during the Indonesia wave. Let me check.

Reply:
👍 see slack